ledfx/presets.py

```diff
@@ -2379,13 +2376,11 @@
                 "frequency_range": "Lows (beat+bass)",
                 "gradient": "linear-gradient(90deg, #00ffff 0.00%,#0000ff 100.00%)",
                 "gradient_name": "Ocean",
-                "gradient": "linear-gradient(90deg, rgb(0, 255, 255) 0%, rgb(0, 0, 255) 100%)",
```

Comment:
I suspect the last one is the one that should be preserved here. although it may be the same values actually, just written differently